bundlewrap/items/git_deploy.py

```diff
@@ -207,6 +207,7 @@ def run_git(self, cmdline, repo_dir):
         """
         git_env = environ.copy()
         git_env['GIT_TERMINAL_PROMPT'] = '0'
+        git_env['GIT_SSH_COMMAND'] = 'ssh -o BatchMode=yes'
```

Comment:
Do we maybe want all of this? https://github.com/bundlewrap/bundlewrap/blob/main/bundlewrap/operations.py#L309-L320

Reply:
Explicitly disabling interactive authentication methods doesnt change the outcome, but it makes sense to not even try those methods in the first place.

StrictHostKeyChecking in my opinion should not be touched here. It should only apply to the node currently being applied.